src/lib/secureStorage.ts

```
@@ -0,0 +1,259 @@
+      for (let i = 0; i < decoded.length; i++) {
+        result += String.fromCharCode(
+          decoded.charCodeAt(i) ^ this.KEY.charCodeAt(i % this.KEY.length)
+        )
+      }
+      return JSON.parse(result)
+    } catch (error) {
+      console.error('Fallback decryption error:', error)
+      return null
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)